build/config/android/config.gni

```diff
@@ -94,20 +94,19 @@ if (is_android) {
 
   # Subdirectories inside android_ndk_root that contain the sysroot for the
   # associated platform.
-  _android_api_level = 16
+  _android_api_level = 24
```

Comment:
I always get these confused, but is api_level the minimum level you can target or the maximum level you can use features from?  This is going from Jellybean (4.1.x) to Nougat (7.0)